package.json

```diff
@@ -26,33 +24,35 @@
     "react-native-image-picker": "^0.28.1",
     "react-native-splash-screen": "^3.2.0",
     "react-native-vector-icons": "^6.5.0",
+    "react-native": "0.59.9",
```

Comment:
Can we stick to alphabetical order for dependencies?

Reply:
Well, it is how sorting extension from VSCode did this.

Reply:
They use different alphabet then ;) If you do `yarn add dependency` it follows the rule, that "react-native" should be before any "react-native-X"